src/index.js

```diff
@@ -12,22 +12,3 @@ document.addEventListener('DOMContentLoaded', () => {
 container.addEventListener('click', (e) => {
   sendLikes(e);
 });
-
-// const addGame = async (url = '', data = {}) => {
-//   const request = await fetch(url, {
-//     method: 'POST',
-//     headers: {
-//       'Content-Type': 'application/json',
-//     },
-//     body: JSON.stringify(data),
-//   });
-//   const response = request.text();
-//   return response;
-// };
-
-// addGame(
-//   'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/',
-//   {
-//     name: 'myNewApp',
-//   }
-// ).then((response) => console.log(response));
```

Comment:
- Its better to remove commented line of old code for better code standard.

Reply:
Oh thanks for pointing that out. I totally forgot to do that :+1: